src/ResourceManager/Relay/Commands.Relay/help/New-AzureRmRelayHybridConnection.md

```diff
@@ -214,30 +214,6 @@ System.String
 
 ### Microsoft.Azure.Commands.Relay.Models.HybridConnectionAttibutes
 
-### Examples - 1 : InputObject
```

Comment:
why remove these examples?

Reply:
@markcowl those are sample outputs for the cmdlet - I cut them, but forgot to paste them into the corresponding example code blocks above. I just pushed that change.